definitions/npm/axios_v0.18.x/flow_v0.94.x-/axios_v0.18.x.js

```diff
@@ -0,0 +1,162 @@
+  }
+  declare type $AxiosXHRConfig<T, R = T> = AxiosXHRConfig<T, R>;
+  declare class AxiosXHR<T, R = T> {
+    config: AxiosXHRConfig<T, R>;
+    data: R;
+    headers?: Object;
+    status: number;
+    statusText: string;
+    request: http$ClientRequest<> | XMLHttpRequest;
+  }
+  declare type $AxiosXHR<T, R = T> = AxiosXHR<T, R>;
+  declare type AxiosInterceptorIdent = number;
+  declare class AxiosRequestInterceptor<T, R = T> {
+    use(
+      successHandler: ?(
+        response: AxiosXHRConfig<T, R>
+      ) => Promise<AxiosXHRConfig<*, *>> | AxiosXHRConfig<*, *>,
+      errorHandler: ?(error: mixed) => mixed
+    ): AxiosInterceptorIdent;
+    eject(ident: AxiosInterceptorIdent): void;
+  }
+  declare class AxiosResponseInterceptor<T, R = T> {
+    use(
+      successHandler: ?(response: AxiosXHR<T, R>) => mixed,
+      errorHandler: ?(error: $AxiosError<any>) => mixed
+    ): AxiosInterceptorIdent;
+    eject(ident: AxiosInterceptorIdent): void;
+  }
+  declare type AxiosPromise<T, R = T> = Promise<AxiosXHR<T, R>>;
+  declare class Axios {
+    <T, R>(
+      config: AxiosXHRConfig<T, R> | string,
+      config?: AxiosXHRConfig<T, R>
+    ): AxiosPromise<T, R>;
+    constructor<T, R>(config?: AxiosXHRConfigBase<T, R>): void;
+    request<T, R>(config: AxiosXHRConfig<T, R>): AxiosPromise<T, R>;
+    delete<T, R>(
+      url: string,
+      config?: AxiosXHRConfigBase<T, R>
+    ): AxiosPromise<T, R>;
+    get<T, R>(
+      url: string,
+      config?: AxiosXHRConfigBase<T, R>
+    ): AxiosPromise<T, R>;
+    head<T, R>(
+      url: string,
+      config?: AxiosXHRConfigBase<T, R>
+    ): AxiosPromise<T, R>;
+    post<T, R>(
+      url: string,
+      data?: mixed,
+      config?: AxiosXHRConfigBase<T, R>
+    ): AxiosPromise<T, R>;
+    put<T, R>(
+      url: string,
+      data?: mixed,
+      config?: AxiosXHRConfigBase<T, R>
+    ): AxiosPromise<T, R>;
+    patch<T, R>(
+      url: string,
+      data?: mixed,
+      config?: AxiosXHRConfigBase<T, R>
+    ): AxiosPromise<T, R>;
+    interceptors: {
+      request: AxiosRequestInterceptor<mixed>,
+      response: AxiosResponseInterceptor<mixed>,
+    };
+    defaults: { headers: Object } & AxiosXHRConfig<*, *>;
+  }
+
+  declare class AxiosError<T, R = T> extends Error {
+    config: AxiosXHRConfig<T, R>;
+    request?: http$ClientRequest<> | XMLHttpRequest;
+    response?: AxiosXHR<T, R>;
+    code?: string;
+  }
+
+  declare type $AxiosError<T, R = T> = AxiosError<T, R>;
+
+  declare interface AxiosExport extends Axios {
+    <T, R>(
+      config: AxiosXHRConfig<T, R> | string,
+      config?: AxiosXHRConfig<T, R>
+    ): AxiosPromise<T, R>;
+    Axios: typeof Axios;
+    Cancel: Class<Cancel>;
+    CancelToken: Class<CancelToken>;
+    isCancel(value: any): boolean;
+    create(config?: AxiosXHRConfigBase<any, any>): Axios;
+    all: typeof Promise.all;
+    spread(callback: Function): (arr: Array<any>) => Function;
+  }
+  declare module.exports: AxiosExport;
+}
```

Comment:
You'll need to also copy the tests, and scope them under the the `axios_v0.18.x/flow_v0.93.x-` folder.

Reply:
I notice `definitions/npm/axios_v0.18.x/test_axios_v0.18.x.js` exists so is it necessary to copy the tests?

Reply:
Clarification: only if they break and requires some changes as well.

Reply:
So let's wait for the CI and see :)

Reply:
Looks like we don't need to copy tests then? :D